sql/rowexec/ddl_iters.go

```diff
@@ -1806,74 +1806,79 @@ func getCheckAlterableTable(t sql.Table) (sql.CheckAlterableTable, error) {
 	}
 }
 
+// generateIndexName generates a unique index name based on the columns in the index, and any existing indexes on the table.
+func generateIndexName(ctx *sql.Context, idxAltable sql.IndexAlterableTable, idxColNames []string) (string, error) {
```

Comment:
the comment above that this is different than Dolt's implementation is maybe informative to keep?

Reply:
I can't find the deleted comment, but I think I might've been mistaken before.